src/components/Button.js

```diff
@@ -137,7 +170,7 @@ const Button = React.createClass({
     return (
       <div {...this.props} style={[styles.component, styles[this.props.type], this.props.style]}>
         {this.props.icon ? <Icon size={20} style={styles.icon} type={this.props.icon} /> : null}
```

Comment:
Hiding the icon seems most appropriate. Which brings up a thought...what if instead of using a circle with borders for the spin icon, we just use the `sync` icon from the Icon component? @derek-boman what say ye?